src/navsat_transform.cpp

```diff
@@ -80,7 +94,7 @@ void NavSatTransform::run()
   double transform_timeout = 0.0;
 
   // Load the parameters we need
-  magnetic_declination_ = node_->get_parameter("magnetic_declination_radians");
+  magnetic_declination_ = node_->declare_parameter("magnetic_declination_radians", 0.0);
```

Comment:
You added the declare parameter to a couple, but you didn't update any of the `get_parameter_or` which don't declare variables, they all need to be updated

Reply:
Fixed!